x/subscription/types/keys.go

```diff
@@ -17,6 +17,22 @@ const (
 	MemStoreKey = "mem_subscription"
 )
 
+const (
+	// SubscriptionKeyPrefix is the prefix to retrieve all Subscription
+	SubscriptionKeyPrefix = "Subscribe/"
```

Comment:
normally we'll use underscores (change to "Subscribe_")

Reply:
Other modules use this too - see x/spec/key_spec.go for example.